py/desisim/templates.py

```diff
@@ -1811,6 +1816,10 @@ def make_templates(self, nmodel=100, zrange=(0.5, 4.0), rmagrange=(20.0, 22.5),
 
             if mag is None:
                 mag = rand.uniform(rmagrange[0], rmagrange[1], nmodel).astype('f4')
+	# Read in the BAL templates if needed. 
+        if bal: 
+            baltemplatefile = "/global/project/projectdirs/desi/spectro/templates/basis_templates/basis_templates_svn/trunk/BAL-templates-v0.1.fits" 
+            balwave, baltemplates = bal.readbaltemplates(baltemplatefile)
```

Comment:
The location of the file definitely shouldn't be hard-coded.  Please use an updated version of `desisim.io.read_basis_templates` which makes use of the `DESI_BASIS_TEMPLATES` environment variable in order to find the underlying basis spectra.

Reply:
I've fixed this.